diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/AbstractClassesRule.kt

```diff
@@ -45,25 +45,9 @@ class AbstractClassesRule(configRules: List<RulesConfig>) : DiktatRule(
         if (functions.isNotEmpty() && functions.none { hasAbstractModifier(it) }) {
             CLASS_SHOULD_NOT_BE_ABSTRACT.warnAndFix(configRules, emitWarn, isFixMode, identifier, node.startOffset, node) {
                 val modList = classNode.getFirstChildWithType(MODIFIER_LIST)!!
-                if (modList.getChildren(null).size > 1) {
-                    val abstractKeyword = modList.getFirstChildWithType(ABSTRACT_KEYWORD)!!
-
-                    // we are deleting one keyword, so we need to delete extra space
-                    val spaceInModifiers = if (abstractKeyword == modList.firstChildNode) {
-                        abstractKeyword.treeNext
-                    } else {
-                        abstractKeyword.treePrev
-                    }
-                    modList.removeChild(abstractKeyword)
-                    if (spaceInModifiers != null && spaceInModifiers.isWhiteSpace()) {
-                        modList.removeChild(spaceInModifiers)
-                    }
-                } else {
-                    if (modList.treeNext.isWhiteSpace()) {
-                        classNode.removeChild(modList.treeNext)
-                    }
-                    classNode.removeChild(modList)
-                }
+                val abstractKeyword = modList.getFirstChildWithType(ABSTRACT_KEYWORD)!!
+                val newOpenKeyword = KotlinParser().createNode("open")
```

Comment:
This change seems simple enough not to invoke KotlinParser. Can we just replace node with `asbstract` with a new `LeafPsiElement`?

Reply:
Yes. Fixed